README.md

```diff
@@ -31,6 +31,11 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | tags | A map of tags to apply to resources. | map | `<map>` | no |
 | vpc_id | VPC ID. | string | - | yes |
 | wd_ecs_task_count | Number of ECS tasks to create. | string | `1` | no |
+| wd_instance_type | Waggle Dance instance type, possible values: ecs,ec2. | string | `ecs` | no |
```

Comment:
possible values: ecs,ec2. --> possible values: `ecs`,`ec2`. (i.e., put the possible values in single backticks in markdown)